### ISSUE TITLE
"Using Google Search"  blocked in chrome

### ISSUE THREAD

**Issue by MiniFullC** [EFForg/privacybadger#2998]
**What is your browser and browser version?**

* chrome dev (128.0.6585.0) + Chrom Web Store

**What is broken and where?**

* When I select any text in the browser, right-click, and use Google Search , I expect it to open https://www.google.com.hk/search?q= in a new tab, but the tab shows that it is blocked by the Privacy Badger extension, like `chrome-extension://pkehgij***/data/web_accessible_resources/redirect.html?url=https://www.google.com.hk/search?*****`

* When I add [www.google.com.hk](http://www.google.com.hk/) to the disabled domains, everything works normally.

* It seems that the reason is that ` .hk ` is being blocked.

**Comment by ghostwords** [comment]
Hello and thanks for opening an issue! Does this happen in Incognito windows by any chance?

**Comment by MiniFullC** [comment]
> Hello and thanks for opening an issue! Does this happen in Incognito windows by any chance?您好，感谢您打开问题！这是否发生在隐身窗口中？

It's not

**Comment by ghostwords** [comment]
Could you post a screenshot of what the blocked tab looks like? Thank you!

**Comment by ghostwords** [comment]
Also, are you able to reproduce if you [switch to a new browser profile in Chrome](https://support.google.com/chrome/answer/2364824) and [install Privacy Badger](https://chromewebstore.google.com/detail/privacy-badger/pkehgijcmpdhfbdbbnkijodmdjhbjlgp) there?

**Comment by MiniFullC** [comment]
![image](https://github.com/user-attachments/assets/819ba9f8-88fc-4bba-9064-65d7276a3280)

**Comment by MiniFullC** [comment]
> Also, are you able to reproduce if you [switch to a new browser profile in Chrome](https://support.google.com/chrome/answer/2364824) and [install Privacy Badger](https://chromewebstore.google.com/detail/privacy-badger/pkehgijcmpdhfbdbbnkijodmdjhbjlgp) there?


It's the same too.

**Comment by ghostwords** [comment]
This looks like a Chrome Dev bug, not yet in Chrome Beta (127.0.6533.57).

**Comment by MiniFullC** [comment]
I'll continue to use it and observe, and if there are any new changes, I'll provide feedback in a timely manner.

**Comment by ghostwords** [comment]
Chromium bug: https://issues.chromium.org/issues/354748637#comment15

**Comment by ghostwords** [comment]
A fix was merged to the 128 branch in Chromium. Thanks again for reporting! Let us know if the problem comes back.

**Comment by Ladish** [comment]
> A fix was merged to the 128 branch in Chromium. Thanks again for reporting! Let us know if the problem comes back.

I'm using Chrome Version 131.0.6778.109 and this issue is still not fixed.
![image](https://github.com/user-attachments/assets/bc2bc09d-714f-4d45-9e51-a5c7b9122d04)

**Comment by ghostwords** [comment]
@Ladish Could you open a new issue that explains how to reproduce the problem? Thank you!
